### PR TITLE
Add Julian to Volunteers

### PR DESCRIPTION
**Pull request by Arvin21M**
WIP

Build Preview:

- [opensats.org/about](https://vercel.live/open-feedback/os-website-git-arvin21m-patch-1-open-sats.vercel.app?via=pr-comment-visit-preview-link&passThrough=1)

Still TODO:
- [x] Verify name/nym
- [x] Add social link yes/no
- [x] Get profile image
- [x] Write "about" text
- [x] Proof-read text

### CODE REVIEW

**Comment by Arvin21M** [comment]
@julian2000-code please take a [look at the build preview](https://vercel.live/open-feedback/os-website-git-arvin21m-patch-1-open-sats.vercel.app?via=pr-comment-visit-preview-link&passThrough=1) and review whether or not you would like any final changes made. 🙏

File: data/authors/julian.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: Julian 
+nym: Dynamic
```
Comment (julian2000-code) [inline data/authors/julian.mdx:3]: let's change the nym to lowercase so that it reads "dynamic"

File: data/authors/julian.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+nostr: npub16h5u7amjy5cwdxhru05mmql5cr954a2cpa2klj3685l72qzxm7wsskwy84
+github: https://github.com/julian2000-code
+volunteer: true
+---
+
+Julian's life experiences taught him the value of self-sovereignty. When he
+found Bitcoin, it opened his eyes to how technology can be used to enhance
+privacy and freedom instead of controlling others. Since then, Julian enjoys
+testing different self-sovereign tools, understanding the technical details
+behind how they work, and implementing his favorite tools into his everyday
```
Comment (julian2000-code) [inline data/authors/julian.mdx:17]: change from "implementing his favorite tools _into_" to "implementing his favorite tools _in_ his everyday life"

**Review by Arvin21M — COMMENTED** [review]
Looks good to go

**Review by julian2000-code — APPROVED** [review]

**Comment by julian2000-code** [comment]
Everything looks good to me. 

I tried approving by going to "File changes" => "Review Changes" => "Approve" => "Submit Review". Let me know if I did that incorrectly.

**Comment by Arvin21M** [comment]
> Let me know if I did that incorrectly.

You did this correctly. 🙏

**Review by dergigi — APPROVED** [review]
Looks good to me ✅

**Pull request merged**